src/datasets/batch_view/addons/reference.md

```diff
@@ -1,64 +1,27 @@
 
-<!-- toc -->
+Addon usage by client, partitioned by day.
 
-# Introduction
+This documentation is a work in progress. Work is
+being tracked in [bug 1364172](https://bugzilla.mozilla.org/show_bug.cgi?id=1364172)
 
-{{#include ./intro.md}}
+## Data Reference
 
-# Data Reference
-
-## Example Queries
-
-## Sampling
-
-It contains one or more records for every
-[Main Summary](../main_summary/reference.md)
-record that contains a non-null value for `client_id`.
+This dataset contains one or more records for every entry in the main ping table
+that contain a non-null value for `client_id`.
 Each Addons record contains info for a single addon,
 or if the main ping did not contain any active addons,
 there will be a row with nulls for all the addon fields
 (to identify `client_id`s/records without any addons).
 
-Like the Main Summary dataset, No attempt is made to de-duplicate submissions by `documentId`, so any analysis that could be affected by duplicate records should take care to remove duplicates using the `documentId` field.
-
 ## Scheduling
 
-This dataset is updated daily via the
-[telemetry-airflow](https://github.com/mozilla/telemetry-airflow) infrastructure.
+This dataset is updated daily via the [telemetry-airflow](https://github.com/mozilla/telemetry-airflow) infrastructure.
 The job DAG runs every day after the Main Summary data has been generated.
-The DAG is [here](https://github.com/mozilla/telemetry-airflow/blob/master/dags/main_summary.py#L36).
+The DAG defined in [`dags/bqetl_addons.py`](https://github.com/mozilla/telemetry-airflow/blob/master/dags/bqetl_addons.py)
 
-## Schema
+## Code Reference
 
-As of 2017-03-16, the current version of the `addons` dataset is `v2`,
- and has a schema as follows:
-```
-root
- |-- document_id: string (nullable = true)
- |-- client_id: string (nullable = true)
- |-- subsession_start_date: string (nullable = true)
- |-- normalized_channel: string (nullable = true)
- |-- addon_id: string (nullable = true)
- |-- blocklisted: boolean (nullable = true)
- |-- name: string (nullable = true)
- |-- user_disabled: boolean (nullable = true)
- |-- app_disabled: boolean (nullable = true)
- |-- version: string (nullable = true)
- |-- scope: integer (nullable = true)
- |-- type: string (nullable = true)
- |-- foreign_install: boolean (nullable = true)
- |-- has_binary_components: boolean (nullable = true)
- |-- install_day: integer (nullable = true)
- |-- update_day: integer (nullable = true)
- |-- signed_state: integer (nullable = true)
- |-- is_system: boolean (nullable = true)
- |-- submission_date_s3: string (nullable = true)
- |-- sample_id: string (nullable = true)
-```
-For more detail on where these fields come from in the
-[raw data](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/environment.html#addons),
-please look
-[in the `AddonsView` code](https://github.com/mozilla/telemetry-batch-view/blob/master/GRAVEYARD.md#main-summary-clients-daily-and-addons).
```

Comment:
The schema definitions were constantly going out of date and probably not that useful to be honest. Maybe we could do better with autogenerated documentation.

Reply:
Or maybe we can add a link to the dataset in the [schema-dictionary](https://mozilla-schema-dictionary.netlify.app/) in the future.

Reply:
Yup, or (thinking slightly more near-term) if we added docs on derived datasets to bigquery-etl, an autogenerated schema definition would be high on my list of things to put in.